SectionedCollectionView/Classes/ItemCollectionViewCell.swift

```diff
@@ -18,6 +18,6 @@ open class ItemCollectionViewCell: UICollectionViewCell {
         return "ItemCollectionViewCell"
     }
     
-    open func configure(withValue value: Any){}
+    open func configure(withValue value: Any?){}
```

Comment:
why is this not implemented? If it is for a a subclass to implement maybe we can indicate it some way? maybe a fatalError is too much, but we can find another option